packages/kbn-flot-charts/lib/jquery_flot.js

```diff
@@ -1,3168 +1,3777 @@
+(function ($) {
+  // A jquery-esque isNumeric method since we currently support 1.4.4
+  // and $.isNumeric was introduced on in 1.7
+  const isNumeric =
+    $.isNumeric ||
+    function (obj) {
+      return obj - parseFloat(obj) >= 0;
+    };
 
-// the actual Flot code
-(function($) {
+  /**
+   * The Canvas object is a wrapper around an HTML5 <canvas> tag.
+   *
+   * @constructor
+   * @param {string} cls List of classes to apply to the canvas.
+   * @param {element} container Element onto which to append the canvas.
+   *
+   * Requiring a container is a little iffy, but unfortunately canvas
+   * operations don't work unless the canvas is attached to the DOM.
+   */
+  function Canvas(cls, container) {
+    let element = container.children('.' + cls)[0];
```

Comment:
eslint must have changed var to let/const throughout this file. oh well, not a big deal.